docs/clusters/blanca/blanca.md

```diff
@@ -258,7 +257,7 @@ module load python
 python myscript.py
 ```
 
-Same as Example 1, but specify desired node features, in this case the avx2 instruction set and Redhat V7 OS (job will run on any node meeting these feature requirements, and which has at least 16 cores per node).
+Same as Example 1, but specify desired node features, in this case the avx2 instruction set and RedHat V7 OS (job will run on any node meeting these feature requirements, and which has at least 16 cores per node).
```

Comment:
Can you remove the part: 
```
The ‘blanca’ QoS: Note that as of 1 March, 2018, the “preemptable” qos replaces the previous low-priority QoS, “blanca”, which is no longer active.
```
![Screenshot 2024-09-18 at 4 16 59 PM](https://github.com/user-attachments/assets/ebdb8ef2-f5ae-423e-87c1-31ac0497ebca)

here